source/index.html

```diff
@@ -25,9 +25,12 @@
         <div class="topbar">
           <!--Not sure if we can apply the hyperlink tag 'a' here-->
           <div class="ml-auto">
-            <a href="#home">Home</a>
-            <a href="#fav">Favorite</a>
-            <a href="#shopList">Shopping List</a>
+            <button id = "home">Home</button>
```

Comment:
Removing `<a href="#home">` and replacing it with `button id="home"` didn't give you any errors when using the buttons on the top right?

Reply:
maybe that's why the home button stopped working?

Reply:
i'll look into it later today!